### ISSUE TITLE
Pop-up in UI after installing helm

### ISSUE THREAD

**Issue by agriii927** [seriohub/velero-helm#36]
Immediately after installing chart via helm on my k8s cluster, I start to get pop ups in UI that something went wrong and there are some problems in JSON. Actually i can't handle the mistake because all the values containing symbols < and > either commented or changed to real appropriate values.

![image](https://github.com/user-attachments/assets/fd2fb1af-7a4e-4d58-a073-49f330782e6f)

I tried to find any additional description of the problem in pod's logs, but even with debug mode I couldn't see any details.

Steps to Reproduce:

1. Install Velero UI using Helm with the following command:
    helm install vui seriohub-velero/vui -n velero-ui --create-namespace
2. Check the installation status using Helm.

  Expected Behavior:
  Velero UI should install successfully without any errors.
  
  Actual Behavior: 
  I had a pop-up in UI with the text "Oops, something went wrong. SyntaxError: Unexpected token '<, "<html> <h"... is not valid JSON"
  
  Kubernetes Version: 1.26.

**Comment by davideserio** [comment]
Hi,

On the login page, there is a button in the bottom left corner that displays some diagnostic information. Can you share this information? remove any sensitive data

**Comment by agriii927** [comment]
After start during 5 mins it has statuses like this:
![image](https://github.com/user-attachments/assets/b3ab1b52-d777-439e-a589-c99c02ed5081)

And after it is like:
![image](https://github.com/user-attachments/assets/ef18650e-49df-4876-8b3b-e7f0061d3890)

**Comment by davideserio** [comment]
You are not using the latest available version. The API indicates version 0.15, but API version 0.1.17 is available ([version info](https://github.com/seriohub/velero-helm/blob/main/components.txt)). 
Can you update HELM repository to see if this resolves the error?

**Comment by agriii927** [comment]
yep, I can try it

**Comment by agriii927** [comment]
![image](https://github.com/user-attachments/assets/609129f5-bd3a-46f3-9608-10377778ba67)

ERROR: Exception in ASGI application
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
File "/usr/local/lib/python3.10/site-packages/uvicorn/protocols/http/httptools_impl.py", line 419, in run_asgi
result = await app( # type: ignore[func-returns-value]
File "/usr/local/lib/python3.10/site-packages/uvicorn/middleware/proxy_headers.py", line 84, in call
return await self.app(scope, receive, send)
File "/usr/local/lib/python3.10/site-packages/fastapi/applications.py", line 1054, in call
await super().call(scope, receive, send)
File "/usr/local/lib/python3.10/site-packages/starlette/applications.py", line 123, in call
await self.middleware_stack(scope, receive, send)
File "/usr/local/lib/python3.10/site-packages/starlette/middleware/errors.py", line 186, in call
raise exc
File "/usr/local/lib/python3.10/site-packages/starlette/middleware/errors.py", line 164, in call
await self.app(scope, receive, _send)
File "/usr/local/lib/python3.10/site-packages/starlette/middleware/base.py", line 189, in call
with collapse_excgroups():
File "/usr/local/lib/python3.10/contextlib.py", line 153, in exit
self.gen.throw(typ, value, traceback)
File "/usr/local/lib/python3.10/site-packages/starlette/_utils.py", line 93, in collapse_excgroups
raise exc
File "/usr/local/lib/python3.10/site-packages/starlette/middleware/base.py", line 191, in call
response = await self.dispatch_func(request, call_next)
File "/app/app.py", line 92, in set_called_endpoint
return await call_next(request)
File "/usr/local/lib/python3.10/site-packages/starlette/middleware/base.py", line 166, in call_next
raise RuntimeError("No response returned.")
RuntimeError: No response returned.

**Comment by davideserio** [comment]
Can you check the API status at https://<your-velero-url>/api ?

Also, can you share the override file so we can replicate the configuration? Remove any sensitive data

**Comment by agriii927** [comment]
velero ui api status is alive

global:
  clusterName: k8s-test
fullnameOverride: velero-ui
uiIngress:
  enabled: true
  ingressClassName: nginx"
  host: velero-ui...
  metadata:
    annotations:
      cert-manager.io/cluster-issuer: cluster-ca
      kubernetes.io/ingress.class: nginx
  spec:
    tls:
      - hosts:
        - velero-ui...
        secretName: velero-ui-tls
uiNp:
  enabled: false
apiConfig:
  debugLevel: debug
  awsAccessKeyId: ..
  awsSecretAccessKey: ..
api:
  apiServer:
    image:
      registry: registry...
      repository: velero/velero-api
ui:
  webServer:
    image:
      registry: registry...
      repository: velero/velero-ui
report:
  successfulJobsHistoryLimit: 2
  veleroWatchdogReport:
    image:
      registry: registry...
      repository: velero/velero-watchdog
watchdogConfig:
  processClusterName: test-cluster
watchdog:
  veleroMonitoring:
    image:
      registry: registry...
      repository: velero/velero-watchdog
      
I've tried to run a new version of a chart but the api fails periodically with python error 


Also I've tried to run it in a new environment of minikube 1.28, but it also doesn't work. I run it without tls enabled so I got errors in browser console of the code 401 Unauthorised while I am trying to authorize with default creds. 

If I do the same with http links (not https, I changed links in values and ui cm) I got errors with websocket 

ws://velero-ui.apps.example.local/ws/auth' failed: WebSocket is closed before the connection is established.

<img width="552" alt="image" src="https://github.com/user-attachments/assets/d214fc8c-89f0-46db-9c3c-c5529a736048">

the chart version is 0.1.13
